lib/alex_koin/slack_commands.ex

```diff
@@ -16,6 +24,19 @@ defmodule AlexKoin.SlackCommands do
       db_user ->
         db_user
     end
+
+    update_user_info(user, user_info)
+  end
+
+  defp update_user_info(user, nil), do: user
+  defp update_user_info(user, user_info) do
+    %{ :profile => %{ :first_name => first_name, :last_name => last_name, :email => email }} = user_info
```

Comment:
I'm seeing arrow notation used here, but atom notation below in your maps. Can we make consistent please?